File: ts/iq/w-biz-msg_feedback~set~user_feedback.ts
/** <iq type="set" xmlns="w:biz:msg_feedback">. Replies with one of: Success, InvalidRequest, ServerError. */
export function buildBizMsgUserFeedbackUpdatePreference(action: string, jid: string, feedback?: string): BinaryNode {
  return { tag: 'iq', attrs: { xmlns: 'w:biz:msg_feedback', to: 's.whatsapp.net', type: 'set' }, content: [
      { tag: 'user_feedback', attrs: { action: action, jid: jid, feedback: feedback } },
    ] }
}

File: rust/iq/w-biz-msg_feedback~set~user_feedback.rs
/// <iq type="set" xmlns="w:biz:msg_feedback">. Replies with one of: Success, InvalidRequest, ServerError.
pub fn build_biz_msg_user_feedback_update_preference(action: String, jid: Jid, feedback: String) -> Node {
    NodeBuilder::new("iq")
        .attr("xmlns", "w:biz:msg_feedback")
        .attr("to", "s.whatsapp.net")
        .attr("type", "set")
        .child(NodeBuilder::new("user_feedback").attr("action", action).attr("jid", jid).attr("feedback", feedback).build())
        .build()
}